modular_skyrat/code/modules/mob/living/carbon/human/species_types/speciessnowflake.dm

```diff
@@ -0,0 +1,95 @@
+	fluff_desc = "Anthropomorphs encompass a lot of the know sapient species by SolGov, and their backgrounds always vary wildly."
+
+/datum/species/jelly
+	femalescreamsounds = list('modular_citadel/sound/voice/scream_f1.ogg', 'modular_citadel/sound/voice/scream_f2.ogg')
+	fluff_desc = "Slimepeople are the result of scientifical experimentation on slimes, which NanoTrasen has heavily invested in. At the current day, most slimepeople are simply consumers of brands of slime genemods sold by NanoTrasen."
+
+/datum/species/plasmaman
+	screamsounds = list('modular_citadel/sound/voice/scream_skeleton.ogg')
+	fluff_desc = "Plasmamen are the result of catastrophic experimentation with plasma. They are very fragile and in most circumstances require external support to survive, and for this reason are considered most of the time simply cheap workforce."
+
+/datum/species/pod
+	femalescreamsounds = list('modular_citadel/sound/voice/scream_f1.ogg', 'modular_citadel/sound/voice/scream_f2.ogg')
+	fluff_desc = "Podpeople are simply plantified clones of other sapient species - No one knows where podplants were found for sure, but podpeople have since formed splinter groups about glorifying and preserving nature, coexisting with advanced technology."
+
+/datum/species/shadow
+	femalescreamsounds = list('modular_citadel/sound/voice/scream_f1.ogg', 'modular_citadel/sound/voice/scream_f2.ogg')
+	fluff_desc = "Shadowpeople are considered ashlanders that were somehow cursed by the necropolis. They are doomed to live in the dark, as light burns their flesh away."
+
+/datum/species/synthliz
+	screamsounds = list('modular_citadel/sound/voice/scream_silicon.ogg')
+	fluff_desc = "Synthetic lizardpeople are simply a variant of IPCs made for the sake of looking more appealing, but suffer with the same problems that IPCs do."
+
+/datum/species/vampire
+	femalescreamsounds = list('modular_citadel/sound/voice/scream_f1.ogg', 'modular_citadel/sound/voice/scream_f2.ogg')
+	fluff_desc = "Vampires are cursed humans, doomed to feast on blood to survive."
+
+/datum/species/vox
+	screamsounds = list('modular_skyrat/sound/emotes/voxscream.ogg')
+	fluff_desc = "Vox are an avian species found around the frontier. They are seem as terrible raiders by most other species, and suffer prejudice for it."
+
+/datum/species/xeno
+	screamsounds = list('sound/voice/hiss6.ogg')
+	fluff_desc = "Xenomorph hybrids are the result of initially failed genetic experimentation with xenos, but since then genemods for \"Xenomorphication\" have become very popular."
+
+/datum/species/zombie
+	screamsounds = list('modular_skyrat/sound/zombie/zombie_scream.ogg')
+	fluff_desc = "Zombies are the result of romerol, a deadly reagent that necrotizzes flesh and leads the infected to extreme rage through the manufacture of nanites. Although sapient zombies exist, they suffer heavy prejudice because of their source."
```

Comment:
"Zombies are the result of an as of yet largely unknown jailbroken Nanomachine software program, which overload the body by using natural metabolic and cardiovascular systems to constantly replicate themselves within a host. Hosts are extremely aggressive on account of neurological changes imposed by the nanomachines, driven to spread their affliction to anyone who isn't infected. While sapient zombies do exist, able to regulate their nanomachines to be nonlethal and overall benign, they still suffer heavy prejudice on account of their origin.

Reply:
changing it